packages/story-editor/src/components/floatingMenu/menus/image.js

```diff
@@ -36,8 +35,6 @@ import {
 const FloatingImageMenu = memo(function FloatingImageMenu() {
   return (
     <>
-      <SwapMedia />
-
       <Separator />
```

Comment:
let's remove the separator here (as well as on video menu)
<img width="125" alt="Screen Shot 2022-03-09 at 5 13 05 PM" src="https://user-images.githubusercontent.com/10720454/157561297-5ce02431-e1bb-4aa8-ad5f-698ed679aa8f.png">

Reply:
🤦🏻‍♀️